app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/DeviceCommandsTest.java

```diff
@@ -244,7 +244,7 @@ public void verify500HTTPStatusCode() {
     }
 
     @Test
-    public void findElementWithContextId() throws JSONException {
+    public void findElementWithContextId1() throws JSONException {
```

Comment:
Can we have better names for tests instead of just 1,2,3...?

Reply:
Different? Yes. Better? Not really.